### PR TITLE
[WR-1535] make high opportunity table responsive

### PR DESCRIPTION
**Pull request by jonathan-longe**
The client uploaded the following image showing a discrepancy between the wireframes and production:

Mobile HOO table

Below is the fix as shown on iPhone:

Screenshot from 2023-08-08 15-10-35

### CODE REVIEW

File: src/web/themes/custom/workbc/templates/views/views-view-table.html.twig [hunk 1]
```diff
@@ -100,7 +100,7 @@
               {% set column_classes = column_classes|merge(['views-field-' ~ field]) %}
             {% endfor %}
           {% endif %}
-          <td{{ column.attributes.addClass(column_classes) }}>
+          <td{{ column.attributes.addClass(column_classes) }} data-label="{{ header[key].content }}">
```
Comment (jonathan-longe) [inline src/web/themes/custom/workbc/templates/views/views-view-table.html.twig:103]: @infojunkie I think this is safe to apply to all tables displayed using `views-view-table.html.twig` but I'd appreciate your thoughts

**Comment by jonathan-longe** [comment]
Note: in the iPhone screenshot above, the field content "Diploma/Certificate" is not wrapped as requested in the wireframe.  Would it be possible to change the content to read "Diploma / Certificate" instead so it wraps as requested?

File: src/web/themes/custom/workbc/templates/views/views-view-table.html.twig (hunk 1 above)
Comment (infojunkie) [reply src/web/themes/custom/workbc/templates/views/views-view-table.html.twig:103]: Do you know if `header[key]` is always present ?

**Comment by infojunkie** [comment]
> Note: in the iPhone screenshot above, the field content "Diploma/Certificate" is not wrapped as requested in the wireframe. Would it be possible to change the content to read "Diploma / Certificate" instead so it wraps as requested?

The string "Diploma/Certificate" is part of government data via SSoT. It does not reside in the CMS. I will bring this up in the ticket.

File: src/web/themes/custom/workbc/templates/views/views-view-table.html.twig (hunk 1 above)
Comment (jonathan-longe) [reply src/web/themes/custom/workbc/templates/views/views-view-table.html.twig]: The preamble of the twig file suggests the header is always present, but I'm not willing to risk it.  I've added a safety.  @infojunkie let me know if you think that's sufficient.

**Pull request merged**